js/index.js

```diff
@@ -9,4 +9,207 @@ menuHamburguer.addEventListener('click', () => {
 document.querySelectorAll('.link-nav').forEach((n) => n.addEventListener('click', () => {
   menuHamburguer.classList.remove('active');
   myNav.classList.remove('active');
-}));
\ No newline at end of file
+}));
+
+const myWorkContainer = document.querySelector('.my-work');
+
+const myCardsContent = [
+
+  {
+    myCardId: 'card-1',
```

Comment:
- [optional] Great job so far. Everything looks perfect. Please kindly ensure that when the pop-up comes up, the user should not be able to scroll through your portfolio unless he/she closes it. 😄